pkg/symbol/symbol.go

```diff
@@ -90,10 +90,7 @@ func (s *Symbolizer) resolveJITedFunctions(pid profiler.PID, locations []*profil
 		if !ok {
 			sym, err := perfMap.Lookup(loc.Address)
 			if err != nil {
-				if !errors.Is(err, perf.ErrNotFound) {
```

Comment:
`Lookup()` only only returns an `ErrNoSymbolFound` error

Reply:
I think the `ErrNoSymbolFound` sentinel error only gets checked here. Let's remove the error if we decide not to check against it.

Reply:
For the record, I think it is a best practice to declare all errors, so leaving `ErrNoSymbolFound` even if nothing catches it at the moment is good

I usually enable the [go-err113](https://github.com/Djarvur/go-err113) linter which disallow dynamic errors.

Reply:
We can declare it, but we don't need to export it. If the identifier is exported, it gives me the sense that it's used outside the package.